src/main/java/net/sf/jsqlparser/parser/ParserKeywordsUtils.java

```diff
@@ -182,6 +182,11 @@ public static List<String> getReservedKeywords(int restriction) {
 
                 , { "NEXTVAL", RESTRICTED_JSQLPARSER }
 
+                , { "AUTO", RESTRICTED_JSQLPARSER }
```

Comment:
Are those really restricted Keywords which must not been used?
Or are those just keywords? Why exactly have those Tokens been added?